canonical-web-and-design/ubuntu.com
# Add feedback links to K8s docs

## Done

- Add a feedback link to the bottom of every page
- update crash-reporting section

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

Feedback section should look something like this:

![image](https://user-images.githubusercontent.com/115290/66039609-436c0900-e50d-11e9-8882-3a78e4ecba8f.png)
